frontend/src/components/HeaderDropdown/HeaderDropdown.css

```diff
@@ -0,0 +1,11 @@
+.headerDropdown-profilePic {
```

Comment:
NitPick: It's bad form to use `camelCase` in CSS. Let's use `kebab-case` instead.